build.gradle

```diff
@@ -145,6 +145,9 @@ dependencies {
     compileOnly fileTree(dir: knnJarDirectory, include: '*.jar')
     api group: 'org.opensearch', name:'opensearch-ml-client', version: "${opensearch_build}"
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
+    // ml-common excluded reflection for runtime so we need to add it by ourselves.
+    // https://github.com/opensearch-project/ml-commons/commit/464bfe34c66d7a729a00dd457f03587ea4e504d9
```

Comment:
add a TODO that states that we need to remove it, Once ML commons fixes their issue.